.github/workflows/linters.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: |
           auth_header="$(git config --local --get http.https://github.com/.extraheader)"
           git submodule sync --recursive
-          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=2
```

Comment:
Why?

Reply:
the sentry_native repo itself has submodules: 
https://github.com/getsentry/sentry-native/tree/master/external